Source/WTF/wtf/FileHandle.h

```diff
@@ -27,6 +27,7 @@
 
 #include <optional>
 #include <wtf/Forward.h>
+#include <wtf/MappedFileData.h>
```

Comment:
Seems a bit of a shame that we include this rather than forward declaring. Would be nicer if only callers that are mapping file data needed to include this header.

Reply:
I will fix that before landing.